docs/source/tutorial.rst

```diff
@@ -468,3 +471,58 @@ of the sample ERR981212, which are located in
     read-based coverage files that will help improve the bins,
     see :ref:`How to create read-based coverage files for genome
     binning <boost_binning>`.
+
+
+Bin refinement
+**************
+
+You can also refine your bins through bioinformatic tools like
+`DAS Tool <https://github.com/cmks/DAS_Tool>`_. DAS Tool
+calculates a set of optimized and non-redundant bins from the
+output of different metagenomic binners. It requires tab separated
+scaffolds-to-bin tables from each metagenomic binner and the contigs in
+fasta format that were used to generate these bins.
+
+.. note::
+
+    If you want to know how to create these scaffolds-to-bin tsv files
+    for DAS Tool, please see :ref:`scaffolds-to-bin tsv files for
+    DAS Tool <scaffolds2bin>`.
+
+For the purpose of the tutorial, we will run DAS Tool with a different
+set of samples. This is because DAS Tool needs a specific quality
+threshold that the previous bins did not yield. Please, download the
+input samples from here
+`DAS_Tool_input <https://drive.google.com/drive/folders/1BxtHEh2sMdPB30Q0mB4iX2sFVeeASDEg?usp=sharing>`_.
+Place this file where you are running this tutorial and decompress it:
+
+.. code-block:: bash
+
+    tar -xvzf das_tool_input.tar.gz
+
+Now that we are all set, go ahead and run :code:`metabiome das_tool`
+command like so:
+
+.. warning::
```

Comment:
Maybe would be better to put his warning before _"Now that we are all set..."_?

Reply:
I agree Juan!! Thanks!!!!!!!!!!!!!!! 
:paintbrush: